tools/ci_build/github/linux/docker/scripts/training/ortmodule/stage1/requirements_torch1.12.1_cu11.6/requirements.txt

```diff
@@ -3,4 +3,6 @@
 torch==1.12.1+cu116
 torchvision==0.13.1+cu116
 torchtext==0.13.1
+# TODO(bmeswani): packaging 22.0 removes support for LegacyVersion leading to errors because transformers 4.4.2 uses LegacyVersion
```

Comment:
just to confirm, it's okay to add comment in the requirements.txt?

Reply:
yes it is
https://pip.pypa.io/en/stable/reference/requirements-file-format/#comments